charts/csi-secret-provider-class/values.yaml

```diff
@@ -0,0 +1,16 @@
+# provider -- specify secret provider
+provider: ~
```

Comment:
why exactly `~` and not `""` ?

Reply:
`~` is `null` in YAML and it is used to indicate that this is per default falsy and needs to be set.

Reply:
![](https://media3.giphy.com/media/d2YVk2ZRuQuqvVlu/giphy.gif?cid=ecf05e477c0d559dc5aaaee2d0248c590bba14da5cd0d060&rid=giphy.gif)